### ISSUE TITLE
Add trained model to docker image?

### ISSUE THREAD

**Issue by wcedmisten** [mtlynch/ingredient-phrase-tagger#80]
Would you be open to adding the trained model to the docker image? Or maybe adding a separate Dockerfile that contains the model trained with the default parameters? The model file is only ~13 MB, so it shouldn't be a huge burden in terms of storing an extra layer, but it would save a significant amount of time not having to train it each time the image is run.

**Comment by mtlynch** [comment]
Yeah, that sounds good to me. I wouldn't want the model under source control, but having it in the Docker image makes sense.

**Comment by hay-kot** [comment]
I'm using the trained model as a part of my self-hosted project [Mealie](https://hay-kot.github.io/mealie/) and have published a [trained model here](https://github.com/mealie-recipes/nlp-model). You can use it apart of the build process with curl or wget, here is how I've integrated it into my build process. 

```dockerfile
# copy CRF++ Binary from crfpp
ENV CRF_MODEL_URL=https://github.com/mealie-recipes/nlp-model/releases/download/v1.0.0/model.crfmodel

# Grab CRF++ Model Release
RUN curl -L0 $CRF_MODEL_URL --output $MEALIE_HOME/mealie/services/parser_services/crfpp/model.crfmodel
```
